R/duckplyr-package.R

```diff
@@ -543,7 +543,7 @@ tidyselect::where
 dplyr_mode <- FALSE
 
 on_load({
-  if (!identical(Sys.getenv("TESTTHAT"), "true")) {
+  if (!identical(Sys.getenv("TESTTHAT"), "true") & is.null(getOption("duckdb.materialize_message"))) {
```

Comment:
Inside an `if()` it's almost always better to use `&&` or `||` :

```suggestion
  if (!identical(Sys.getenv("TESTTHAT"), "true") && is.null(getOption("duckdb.materialize_message"))) {
```

Reply:
Okay, I adjusted my change accordingly